### ISSUE TITLE
Tests are failing on rakudo 2020.02

### ISSUE THREAD

**Issue by Altai-man** [zostay/p6-Path-Router#6]
Logs:
```
===> Testing: Path::Router:ver<0.5.0>
[Path::Router] Cannot do 'say' on a handle in binary mode
[Path::Router]   in sub proclaim at /home/koto/.rakudobrew/versions/moar-2020.02/install/share/perl6/core/sources/2D4A7CA10695CD2B374573413D261A660E282E2A (Test) line 735
[Path::Router]   in sub ok at /home/koto/.rakudobrew/versions/moar-2020.02/install/share/perl6/core/sources/2D4A7CA10695CD2B374573413D261A660E282E2A (Test) line 130
[Path::Router]   in sub mapping-is at /home/koto/.zef/store/Path-Router-0.5.0.tar.gz/Path-Router-0.5.0/lib/Test/Path/Router.pm6 (Test::Path::Router) line 115
[Path::Router]   in block  at t/014_test_path_router.t line 137
[Path::Router]   in block <unit> at t/014_test_path_router.t line 121
===> Testing [FAIL]: Path::Router:ver<0.5.0>
```

**Comment by zostay** [comment]
I can reproduce it, but I don't have time to dig into it just now. I suspect something with how I'm using IO::String is a problem. I feel like maybe I need to get rid of the IO::String stuff.

**Comment by zostay** [comment]
I've reported a bug upstream in IO::String, but I'm not sure Rob is doing much Raku these days. In any case, the dependency on IO::String is pretty unnecessary. I rewrote the test to just use a couple temp files for each test. A new release is coming shortly.

I forgot to note in the commit message so this would get updated, but this should be fixed in f4be4d6645

**Comment by zostay** [comment]
For reference the issue report in IO::String is here:

https://github.com/hoelzro/p6-io-string/issues/12
